vulnerable-target-list.json

```diff
@@ -1,3 +1,31 @@
 {
-  "targets": []
-}
+  "targets": [
```

Comment:
"Thanks for the PR! I really like the idea of adding those additional props. Could you also update the expected JSON structure in the README?"